VultisigApp/VultisigApp/Views/Keygen/PeerDiscoveryView.swift

```diff
@@ -61,7 +61,11 @@ struct PeerDiscoveryView: View {
             // only show the QR share button when it is in peer discovery
             if viewModel.status == .WaitingForDevices {
                 ToolbarItem(placement: Placement.topBarTrailing.getPlacement()) {
-                    NavigationQRShareButton(title: "joinKeygen", renderedImage: shareSheetViewModel.renderedImage)
+                    NavigationQRShareButton(
+                        vault: vault, 
```

Comment:
Keep in mind , during keygen , `vault` still don't have public key ecdsa and public key eddsa, please make sure it still works in that scenario